src/_sass/main.scss

```diff
@@ -0,0 +1,3 @@
+@charset "utf-8";
+
+@import "node_modules/bulma/bulma";
```

Comment:
I have a mild preference for continuing to use tailwind instead of adding another css framework to the mix; as I only have so much brain-space for libraries.

I'm content leaving them diverging for now; and would hope we can pick one after a few weeks/months.